tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -0,0 +1,379 @@
+    {
+      let cut = is_set(data.wire_bitflags, i);
+      i++;
+      wireContent.push(
+        <Flex textColor={data.wires[wire]} px={2}>
+          <Flex.Item grow bold>
+            {wire}
+          </Flex.Item>
+          <Flex.Item mr="5%">
+            <Button
+              ml="10%"
+              my="1%"
+              content="Pulse"
+              onClick={() => act('wire', { action: "pulse", wire: i })}
+            />
+          </Flex.Item>
+          <Flex.Item>
+            <Button
+              m="1%"
+              content={cut ? "Cut" : "Mend"}
+              onClick={() => act("wire", { action: (cut ? "mend" : "cut"), wire: i })}
+            />
+          </Flex.Item>
+        </Flex>
+      );
+    }
+  }
+  return (
+    <Collapsible
+      mb="1%"
+      title="Maintenence Panel"
+      open={data.panel_open}
+    >
+      <Box backgroundColor={backgroundPop} p={1}>
+        {wireContent}
+        <Divider />
+        <LabeledList>
+          <LabeledList.Item
+            label="Electrification Risk">
+            {data.indicators.electrified ? "High" : "None"}
+          </LabeledList.Item>
+          <LabeledList.Item
+            label="System Stability">
+            {data.indicators.malfunctioning ? "100%" : "0%"}
+          </LabeledList.Item>
+          <LabeledList.Item
+            label="Warranty">
+            {data.indicators.hacked ? "Void" : "Valid"}
+          </LabeledList.Item>
+          <LabeledList.Item
+            label="Power">
+            {data.indicators.hasPower ? "Sufficient" : "Insufficient"}
+          </LabeledList.Item>
+        </LabeledList>
+      </Box>
+    </Collapsible>
+  );
+};
+
+export const Manufacturer = (_, context) => {
+  const { act, data } = useBackend<ManufacturerData>(context);
+  const [repeat, toggleRepeatVar] = useSharedState(context, "repeat", data.repeat);
+  const [speed, setSpeedVar] = useSharedState(context, "speed", data.speed);
+  let toggleRepeat = () => {
+    act("repeat");
+    toggleRepeatVar(!repeat);
+  };
+  let updateSpeed = (newValue:number) => {
+    act("speed", { "value": newValue });
+    setSpeedVar(newValue);
+  };
+  let usable_blueprints = data.available_blueprints;
+  let dropdowns = [];
+  for (let i of data.all_categories) {
+    if (!usable_blueprints[i]) continue;
+    dropdowns.push(<CategoryDropdown category={i} blueprints={usable_blueprints[i]} />);
+  }
+  return (
+    <Window width={1200} height={600} title={data.fabricator_name}>
+      <Window.Content>
+        <Flex>
+          <Flex.Item
+            style={{ "width": "80%" }}
+            mr={1}
+          >
+            <Section>
+              {dropdowns}
+            </Section>
+          </Flex.Item>
+          <Flex.Item
+            style={{ "width": "20%" }}
+          >
+            <Section
+              fill
+              align="center"
+            >
+              <Input placeholder={"Search..."} icon={"search"} style={{ "width": "100%" }} />
```

Comment:
```suggestion
              <Input placeholder="Search..." icon="search" width="100%" />
```
(Pretty sure `width` works fine that way for `Input`s.)